lib/screens/admin_order_detail_page.dart

```diff
@@ -0,0 +1,585 @@
+                  style: TextStyle(
+                    fontSize: 16,
+                    fontWeight: FontWeight.bold,
+                    color: paletteBlackColor,
+                  ),
+                ),
+              ),
+              Text(
+                ' 点',
+                style: TextStyle(
+                  fontSize: 16,
+                  fontWeight: FontWeight.w300,
+                  color: paletteBlackColor,
+                ),
+              )
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildOrderPriceDescriptionBar() {
+    return Container(
+      padding: EdgeInsets.symmetric( horizontal: 16.0 ),
+      height: 40,
+      child: Container(
+        color: kPaletteWhite,
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: <Widget>[
+            Container(
+              child: Text(
+                '総合計',
+                style: TextStyle(
+                  color: paletteBlackColor,
+                  fontSize: 16,
+                  fontWeight: FontWeight.w600,
+                ),
+              ),
+            ),
+            Container(
+              child: Text(
+                "¥" + NumberFormat("#,##0").format( widget.order.getOrderPrice() ),
+                style: TextStyle(
+                  color: paletteDarkRedColor,
+                  fontSize: 16,
+                  fontWeight: FontWeight.w600,
+                ),
+              ),
+            ),
+          ],
+        )
+      ),
+    );
+  }
+
+  Widget _buildBuyerDetails() {
+    return Container(
+      decoration: BoxDecoration(
+        color: kPaletteWhite,
+        borderRadius: BorderRadius.only(
+          topLeft: Radius.circular(40),
+          topRight: Radius.circular(40),
+        ),
+      ),
+      child: Column(
+        children: <Widget>[
+          _buildGreyBar(),
+          _buildBuyerContactInfoBox(),
+          _buildShippingAddressInfoBox(),
+          _buildShippingMethodInfoBox(),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildGreyBar() {
+    return Container(
+      padding: EdgeInsets.only( top: 24.0 ),
+        child: Container(
+        decoration: BoxDecoration(
+          color: const Color(0xFFEFEFEF),
+          borderRadius: BorderRadius.all( Radius.circular(40) ),
+        ),
+        height: 8,
+        width: 115,
+      ),
+    );
+
+  }
+
+  Widget _buildBuyerContactInfoBox() {
+    return Container(
+      padding: EdgeInsets.only(
+        top: 12.0, left: 24.0, right: 24.0, bottom: 36.0
+      ),
+      child: StreamBuilder<DocumentSnapshot>(
+        stream: Firestore.instance
+            .collection(constants.DBCollections.users)
+            .document(widget.order.customerId)
+            .snapshots(),
+        builder: (context, snapshot) {
+          if (!snapshot.hasData) {
+            return LinearProgressIndicator();
+          }
+          Customer buyer = Customer.fromSnapshot(snapshot.data);
```

Comment:
maybe rename this to `_customer` to be consistent in our naming everywhere? Also note the `_` to make it a private variable